<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>initFile</class>
 <widget class="QWidget" name="initFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>380</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;
border-radius:7px;
padding:2px 4px;
font: 10pt &quot;华文隶书&quot;;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PORT">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>160</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;
border-radius:7px;
padding:2px 4px;
font: 10pt &quot;华文隶书&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>请输入端口号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>470</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;字心坊猫的天空之城（特别定制版）&quot;;</string>
   </property>
   <property name="text">
    <string>停车场名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>40</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;字心坊猫的天空之城（特别定制版）&quot;;</string>
   </property>
   <property name="text">
    <string>停车场数据初始化</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="database">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>230</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;
border-radius:7px;
padding:2px 4px;
font: 10pt &quot;华文隶书&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>请输入数据库名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="account">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>300</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;
border-radius:7px;
padding:2px 4px;
font: 10pt &quot;华文隶书&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>请输入账户</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>540</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;字心坊猫的天空之城（特别定制版）&quot;;</string>
   </property>
   <property name="text">
    <string>停车场价格</string>
   </property>
  </widget>
  <widget class="QPushButton" name="submit_init">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>480</y>
     <width>131</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 170, 127);
color: rgb(255, 255, 255);
font: 25 9pt &quot;Bahnschrift Light&quot;;
font: 10pt &quot;华文隶书&quot;;
border-radius:7px;</string>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="IP">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>90</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;
border-radius:7px;
padding:2px 4px;
font: 10pt &quot;华文隶书&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>请输入IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>240</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;字心坊猫的天空之城（特别定制版）&quot;;</string>
   </property>
   <property name="text">
    <string>数据库名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;字心坊猫的天空之城（特别定制版）&quot;;</string>
   </property>
   <property name="text">
    <string>账户</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;字心坊猫的天空之城（特别定制版）&quot;;</string>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;字心坊猫的天空之城（特别定制版）&quot;;</string>
   </property>
   <property name="text">
    <string>密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="parking_name">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>460</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;
border-radius:7px;
padding:2px 4px;
font: 10pt &quot;华文隶书&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>请输入停车场名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="park_fee">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>540</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;
border-radius:7px;
padding:2px 4px;
font: 10pt &quot;华文隶书&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>请输入停车场 单价每小时</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;字心坊猫的天空之城（特别定制版）&quot;;</string>
   </property>
   <property name="text">
    <string>端口号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="park_count">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>370</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(247, 247, 247);
border:1px groove gray;
border-radius:7px;
padding:2px 4px;
font: 10pt &quot;华文隶书&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>停车场总容量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>330</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;字心坊猫的天空之城（特别定制版）&quot;;</string>
   </property>
   <property name="text">
    <string>停车场容量</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
